uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Service/DrupalUserService.php

```diff
@@ -64,6 +75,8 @@ public function sendActivationMail($email) {
   }
 
   /**
+   * Adds role(s) to user account
```

Comment:
Would be a good idea to update this method to reflect that it adds roles in addition to existing user roles.